lib/rules/web/admin_console/4.2/filters.xyaml

```diff
@@ -1,7 +1,7 @@
 set_filter_strings:
   elements:
   - selector: &filter_text_ele
-      css: input.text-filter
+      css: input[class*='text-filter']
```

Comment:
could you explain how this selector differs from the previous one?

Reply:
bump, this is most interesting one, lgtm otherwise

Reply:
What I see that `*=` means that it will match a class of `something text-filter anotherthing` as well `whaever-text-filter-whatever`.

Reply:
Why is this needed? What class we can't match with original version?

Reply:
Comment added in https://github.com/openshift/verification-tests/pull/299#discussion_r308069274

Reply:
```
<input autocapitalize="none" class="pf-c-form-control co-text-filter" placeholder="Filter Events by name or message..." tabindex="0" type="text" value="">
```

Reply:
```
$$("input[class*='text-filter']")
Array [ input.pf-c-form-control.co-text-filter
 ]

$$('input.text-filter')
Array []
```

Reply:
> What I see that `*=` means that it will match a class of `something text-filter anotherthing` as well `whaever-text-filter-whatever`.

Yes